pages/index.vue

```diff
@@ -165,7 +247,45 @@ export default defineComponent({
      * @todo 1. Send the request to delete the task to the backend server.
      * @todo 2. Remove the task from the dom.
      */
-    deleteTask(_index, _id) {},
+    /*async*/ deleteTask(_index, _id) {
+      let token = this.$store.getters.token;
+      let all = this;
```

Comment:
nice 😄 we generally call it `that` the variable but this also works

Reply:
ohh okay :) I'll do that next timw